.cico/cico_openshift_e2e.sh

```diff
@@ -43,9 +55,24 @@ if ! hash operator-sdk 2>/dev/null; then
         rm operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
 fi
 
+# Function to get all logs and events from devworkspace operator deployments
+function getDevWorkspaceOperatorLogs() {
```

Comment:
It's strange to have a function declaration this far down in the file (after some logic that is executed every time.